jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/config/PrefixSuffixIT.java

```diff
@@ -28,53 +29,74 @@ public void defaultClassPrefix() throws ClassNotFoundException {
         resultsClassLoader.loadClass("com.example.PrimitivePropertiesDao");
     }
-    
+
     @Test(expected = ClassNotFoundException.class)
-    public void NotExitstingClassPrefix() throws ClassNotFoundException{
+    public void NotExistingClassPrefix() throws ClassNotFoundException{
 
         ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitiveProperties.json", "com.example", config("classNamePrefix","Abstract"));
         resultsClassLoader.loadClass("com.example.NotExistingPrimitiveProperties");
     }
-    
+
     @Test(expected = ClassNotFoundException.class)
-    public void NotExitstingClassSufix() throws ClassNotFoundException{
+    public void NotExistingClassSufix() throws ClassNotFoundException{
 
         ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitiveProperties.json", "com.example", config("classNameSuffix","Dao"));
         resultsClassLoader.loadClass("com.example.NotExistingPrimitiveProperties");
     }
-    
+
     @Test(expected = ClassNotFoundException.class)
     public void SuffixWithDefaultPackageName() throws ClassNotFoundException{
         ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitiveProperties.json", "", config("classNameSuffix","Dao"));
         resultsClassLoader.loadClass("com.example.NotExistingPrimitiveProperties");
     }
-    
+
     @Test(expected = ClassNotFoundException.class)
     public void PrefixWithDefaultPackageName() throws ClassNotFoundException{
         ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitiveProperties.json", "", config("classNamePrefix","Abstract"));
         resultsClassLoader.loadClass("com.example.NotExistingPrimitiveProperties");
     }
 
-    
+    @Test
+    public void customClassPrefixNoJavaType() throws ClassNotFoundException {
+
+        ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitivePropertiesNoJavaType.json",
+                "com.example", config("classNamePrefix","Prefix"));
+        resultsClassLoader.loadClass("com.example.PrefixprimitivePropertiesNoJavaType");
+    }
+
+    @Test
+    public void customClassSuffixNoJavaType() throws ClassNotFoundException {
+
+        ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitivePropertiesNoJavaType.json",
+                "com.example", config("classNameSuffix","Suffix"));
+        resultsClassLoader.loadClass("com.example.PrimitivePropertiesNoJavaTypeSuffix");
+    }
+
+    @Test
+    public void customClassPrefixAndSuffixNoJavaType() throws ClassNotFoundException {
+        ClassLoader resultsClassLoader = generateAndCompile("/schema/properties/primitivePropertiesNoJavaType.json",
+                "com.example", config("classNamePrefix", "prefix", "classNameSuffix","Suffix"));
+        resultsClassLoader.loadClass("com.example.PrefixprimitivePropertiesNoJavaTypeSuffix");
+    }
```

Comment:
It doesn't look like the existing functionality does anything to the prefix and suffix, so a test to confirm the existing functionality and a test to make sure the new functionality matches would be good.  Perhaps check `classNamePrefix ` = `prefix` and `classNameSuffix` = `suffix`, expecting `com.example.prefixPrimitivePropertiesNoJavaTypesuffix`.